### PR TITLE
Update py-regions to v0.3

### PR DESCRIPTION
**Pull request by cdeil**
This PR updates `py-regions` to v0.3.

https://pypi.org/project/regions/0.3/

I didn't do any tests locally, I no longer have Macports installed.

### CODE REVIEW

File: python/py-regions/Portfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ set _name           regions
 set _n              [string index ${_name} 0]
 
 name                py-${_name}
-version             0.2
+version             0.3
 categories-append   science
 platforms           darwin
 maintainers         {gmail.com:Deil.Christoph @cdeil} openmaintainer
```
Comment (pmetzger) [inline python/py-regions/Portfile]: If you can no longer maintain this, you should remove yourself as the maintainer as well.

Comment (cdeil) [reply python/py-regions/Portfile]: I'll try to maintain this one and at least update it for new versions like I did here, it's an important one / dependency for other packages.

**Comment by reneeotten** [comment]
You should probably update the dependencies and add ```py-astropy-healpix```, at least that's what is listed in setup.py. Also currently the livecheck does not work, you can just use the default PyPI that is set when using pypi for the master_sites. In other words, just remove the else statement at the end of the Portfile.

**Comment by cdeil** [comment]
@reneeotten - Thank you for the review. Implemented your suggestions in e530417c3d .

**Comment by pmetzger** [comment]
You probably want to squash these two commits...

**Comment by cdeil** [comment]
> You probably want to squash these two commits...

Is leaving as-is an option, or you squashing the commits when you merge this in?
If no, sure, I can do it?
Is there some tag I should put in the commit message to avoid triggering another CI build?

**Comment by pmetzger** [comment]
Merged. Thanks, @cdeil!

**Pull request merged**